assets/css/10ten.css

```diff
@@ -22,6 +44,94 @@ main { min-height: 80vh; }
   }
 }
 
+/* Headings */
+
+h3,
+h4,
+h5 {
+    font-weight: 300;
+    letter-spacing: 1px;
+    color:#444;
+}
+
+h3 {
+    color:#444;
+    font-size: 180%;
+    word-spacing: 2px;
+    text-align: center;
+    margin-bottom: 30px;
+}
+
+h3:after {
+    display: block;
+    height: 2px;
+    background-color: #ff4faa;
+    content: " ";
+    width: 100px;
+    margin: 0 auto;
+    margin-top: 30px;
+}
+
+
+h4 {
+    color:#444;
+    font-size: 125%;
+    word-spacing: 2px;
+    margin-bottom: 30px;
+    text-align: center;
+}
+
+h5 {
+    color:#444;
+    font-size: 120%;
+    margin-bottom: 15px;
+    text-transform: uppercase;
+}
+
+
+/* Paragraphs */
+.long-copy {
+    line-height: 145%;
+    width: 70%;
+    margin-left: 15%;
+}
+
+.box p {
+    font-size: 90%;
+    line-height: 145%;
+}
+
+
+/* Headers */
+header {
+    background-image: linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7));
+    background-size: cover;
+    background-position: top;
+    width: 100%;
+    height: 65vh;
+    margin-top: 2em;
+    margin-bottom: 0;
+    
+}
```

Comment:
this will be applied to all 10ten headers?

Reply:
Yep.. we are calling HEADER* the "Hero area" with the background image, just below the MENU, existent only in the HowItWorks and BecomeAnInstructor pages (there is no HEADER in the Index page for example..). There should be a minimum margin top for the text and a small margin between the button. 
(*) I don't know if this is right... what is the Header? Is it different from the Menu/Nav Bar ?